my-app/src/services/auth.service.js

```diff
@@ -0,0 +1,51 @@
+import Constants from "../constants/constants.service";
+import cookieService from "./cookie.service";
+
+const baseUrl = Constants.getBaseUrl();
+const USER_TOKEN = "USERTOKEN";
+
+const login = (data) => {
+  const options = {
+    method: "POST",
+    headers: {
+      "Content-Type": "application/json",
+    },
+    body: JSON.stringify(data),
+  };
+
+  return fetch(`${baseUrl}/login`, options)
+    .then((res) => {
+      if (!res.ok) {
+        throw new Error(res.statusText);
+      }
+
+      return res.json();
+    })
+    .then((res) => {
+      // save to cookie
+      cookieService.setCookie(USER_TOKEN, res.USERTOKEN);
```

Comment:
Why do you use USER_TOKEN as constant, but not from any type of store?

Reply:
Do you mean to put USER_TOKEN in constantsService, because we don't use any type of storage.

Reply:
I ment here, that i suppose you should get token from localStorage but not from var above)